api/MediaEncryptedEvent.json

```diff
@@ -0,0 +1,209 @@
+{
+  "api": {
+    "MediaEncryptedEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioWorkletProcessor",
```

Comment:
None of these URLs seem to be right. `AudioWorkletProcessor`?

Reply:
Cut and paste error. Sorry about that. PTAL